sources/packages/api/src/route-controllers/institution/institution.controller.ts

```diff
@@ -522,4 +560,124 @@ export class InstitutionController extends BaseController {
       operatingName: institutionDetail.operatingName,
     };
   }
+
+  /**
+   * Controller method to get all institution locations with the
+   * given institutionId for  ministry user.
+   * @param institutionId institution id
+   * @returns All the institution locations for the given institution.
+   */
+  @AllowAuthorizedParty(AuthorizedParties.aest)
+  @Get("/:institutionId/location-summary")
+  async getAllInstitutionLocationSummary(
```

Comment:
can we please rename it to getAllInstitutionLocationSummaryForAEST? I believe, no one other than the ministry user will be using this route